edition/rd

```diff
@@ -6,6 +6,7 @@ LIMA_INSTALL_CNI_PLUGIN_FLANNEL=true
 LIMA_INSTALL_CURL=true
 LIMA_INSTALL_DOCKER=true
 LIMA_INSTALL_CRI_DOCKERD=true
+LIMA_INSTALL_GIT=true
```

Comment:
Could you update other edition files too?
I guess we can safely install git for all editions (except `min`)

cc @mook-as

Reply:
Should now be done in https://github.com/lima-vm/alpine-lima/pull/84/commits/8acde0f04ec55c40ba3a5905c08490ccb9df2de3

Reply:
Apologies for the confusion, not sure why the previous commit didn't get a signed-off-by, but I've now re-pushed it with a signature at 00ab4033ee737c0dc8c7a50383c1f1987b7b25ae